services/cd-service/pkg/repository/transformer_test.go

```diff
@@ -1674,6 +1674,210 @@ func TestApplicationDeploymentEvent(t *testing.T) {
 		})
 	}
 }
+func TestNextAndPreviousCommitCreation(t *testing.T) {
+	type TestCase struct {
+		Name            string
+		Transformers    []Transformer
+		expectedContent []FileWithContent
+		ExpectedError   string
```

Comment:
expected error should be an `error`, not a `string`, see https://github.com/freiheit-com/kuberpult/pull/1433/files